pool/web/view/page/page_contact_experiment.ml

```diff
@@ -189,24 +183,44 @@ let show
         [ h2
             ~a:[ a_class [ "heading-2" ] ]
             [ txt (Utils.nav_link_to_string language I18n.Sessions) ]
-        ; p
-            [ txt (Utils.hint_to_string language I18n.ExperimentSessionsPublic)
-            ]
+        ; p [ txt (hint_to_string I18n.ExperimentSessionsPublic) ]
         ; div [ PageSession.public_overview sessions experiment language ]
         ]
   in
   let waiting_list_form () =
+    let form_action =
+      Format.asprintf
+        "/experiments/%s/waiting-list/"
+        (experiment.Public.id |> Experiment.Id.value)
+      |> (fun url ->
+           if user_is_enlisted then Format.asprintf "%s/remove" url else url)
+      |> Sihl.Web.externalize_path
+    in
+    let text_blocks =
+      let base =
+        if user_is_enlisted
+        then p [ txt (hint_to_string I18n.ContactOnWaitingList) ]
+        else p [ txt (hint_to_string I18n.SignUpForWaitingList) ]
+      in
+      let missing_phone =
```

Comment:
May handle `missing_phone` as an array and use list concat for the combined div. So we don't need the empty `txt`